perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -495,6 +510,33 @@ sub fetch_contacts : Private {
     return 1;
 }
 
+sub fetch_languages : Private {
+    my ( $self, $c ) = @_;
+
+    my $lang_map = {};
+    foreach my $lang (@{$c->cobrand->languages}) {
```

Comment:
Could we have a sort of some sort, each refresh of admin page is changing my order :)